app/routes/settings/applications.js

```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
```

Comment:
Mix `AuthenticatedRouteMixin` with this so that none of the sub-routes are accessible without logging in.

(For ref see `app/routes/events.js`)